### PR TITLE
Correção de responsividade

### PR DESCRIPTION
**Pull request by acrisiodejesus**

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -15,37 +15,37 @@ function Header() {
...
           alt='Mozdevz'
         />
       </Link>
-      <div onClick={handleClick} className='nav-icon'>
+      <NavIcon onClick={handleClick}>
         {open ? <FiX /> : <FiMenu />}
-      </div>
-      <ul className={open ? 'nav-links active' : 'nav-links'}>
+      </NavIcon>
+      <NavList active={open} >
```
Comment (rnrnshn) [inline src/components/Header/index.js:31]: Amazing... good refactoring

File: src/components/Header/index.js [hunk 2]
```diff
@@ -15,37 +15,37 @@ function Header() {
...
         <li className='nav-item'>
-          <Link to='/programas' className='nav-link' onClick={closeMenu}>
+          <NavLink to='/programas' onClick={closeMenu}>
             Programas
-          </Link>
+          </NavLink>
         </li>
         <li className='nav-item'>
-          <Link to='/contacto' className='nav-link' onClick={closeMenu}>
+          <NavLink to='/contacto' className='nav-link' onClick={closeMenu}>
```
Comment (rnrnshn) [inline src/components/Header/index.js]: Why did you keep the `className='nav-link'` on this line??? Isnt the `<NavLink>` a styled component already??

File: src/components/Header/styles.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { css } from "styled-components";
+
+const white = css`
+var(--white);
+`
+
+export const Container = styled.nav`
```
Comment (rnrnshn) [inline src/components/Header/styles.js]: I see u using this Container styled component a lot along with some components above. I understand it is a nav element but I think you should rename it for code readability purposes... Maybe NavContainer?

**Review by rnrnshn — COMMENTED** [review]
The code was perfectly refactored to Styled Component. Please analyze my review and see if it makes sense for you or not then tell me your point of view so we can agree to approve your PR. Thanks for your contribuition

File: src/components/Header/styles.js (hunk 3 above)
Comment (rnrnshn) [reply src/components/Header/styles.js]: And if you don't mind please use commit messages in English and present tense verbs... for example "refactor code to styled-components" or "correct responsiveness" but if language is a barrier for you no problem do the way you are used to

Comment (acrisiodejesus) [reply src/components/Header/styles.js]: Okay, I will follow this tips on my next PR. I agree with you on the question about name of the style components.

File: src/components/Header/index.js (hunk 2 above)
Comment (acrisiodejesus) [reply src/components/Header/index.js]: Ups. I forgot to remove, It shouldn't be there

File: src/components/Header/styles.js (hunk 3 above)
Comment (rnrnshn) [reply src/components/Header/styles.js]: Thanks.... I am still reviewing your PR. So far I like what I see. 🤲🏻

**Review by rnrnshn — APPROVED** [review]
Approved

**Pull request merged**